apps/contract/scripts/deploy.sh

```
@@ -1,43 +1,43 @@
-echo "Contract ID: $CONTRACT_ID"
-
-# Save deployment info to a file
-echo "Saving deployment information..."
-cat > deployment-info.txt << EOF
-Network: $NETWORK
-WASM Hash: $WASM_HASH
-Contract ID: $CONTRACT_ID
-Deployment Date: $(date)
-EOF
-
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)